lessons/intro/distribution/index.md

```diff
@@ -33,38 +33,44 @@ aby mezi nimi nedošlo v textu záměně.
 Anglické pojmy v závorce jsou převzaty z oficiálního [glosáře](https://packaging.python.org/glossary).
 
 * **(importovatelný) modul** (_Module_ ∪ _Import Package_) je cokoliv,
-  co se dá importovat z Pythonu, v tomto textu tedy především Python soubor nebo adresář s nimi;
-* **balíček** (_Distribution Package_) je instalovatelný archiv obsahují
+  co se dá importovat z Pythonu, v tomto textu tedy především Python soubor nebo adresář s vícero Python soubory;
+* **balíček** (_Distribution Package_) je instalovatelný archiv obsahující
   _importovatelné moduly_ pro Python a další potřebné soubory, může být i rozbalený;
-* **zdrojový balíček** (_Source Distribution_, `sdsit`) je varianta zabaleného _balíčku_ ve zdrojové formě;
-* **binární balíček** (_Binary Distribution_, `bdsit`) je varianta zabaleného _balíčku_ v nezdrojové (např. zkompilované) formě;
+* **zdrojový balíček** (_Source Distribution_, `sdist`) je varianta zabaleného _balíčku_ ve zdrojové formě;
+* **binární balíček** (_Binary Distribution_, `bdist`, `wheel`) je varianta zabaleného _balíčku_ v nezdrojové (např. zkompilované) formě;
 * **projekt** (_Project_) je knihovna, framework, skript, plugin, aplikace apod. (či jejich kombinace), které balíme do _balíčků_.
```

Comment:
TODO: přidat build backend do slovníčku

Reply:
Hm, jak přeložit _build backend_ do češtiny? Případně ponechat anglickou verzi?

Reply:
Jednoznačně *postavit zezadu* :D

Nechal bych anglickou verzi.

Reply:
[74c4273](https://github.com/pyvec/naucse-python/pull/29/commits/74c427393b99733eb97ac98431714d689729d0e3)